### PR TITLE
Oct 128

### PR DESCRIPTION
**Pull request by karmseveeratjisc**
The purpose of this PR was...

---

### Acceptance Criteria:

---

### Tests:

---

### CODE REVIEW

File: api/prisma/seed.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import * as client from '../src/lib/client';
 
 export const initialDevSeed = async (): Promise<void> => {
     // Create users
-    await client.prisma.user.createMany({ data: SeedData.usersDevSeedData });
+    await client.prisma.user.createMany({ data: SeedData.usersProdSeedData });
```
Comment (karmseveeratjisc) [inline api/prisma/seed.ts:9]: this is temporary swap

File: api/prisma/seed.ts [hunk 2]
```diff
@@ -18,36 +18,9 @@ export const initialDevSeed = async (): Promise<void> => {
...
-                    currentStatus: publication.currentStatus,
-                    publishedDate: publication.publishedDate,
-                    cleanContent: htmlToText.convert(publication.content)
-                }
-            });
-        }
-    }
 
-    for (const problem of SeedData.problems) {
+    for (const problem of SeedData.problemsProd) {
```
Comment (karmseveeratjisc) [inline api/prisma/seed.ts:23]: again this is temporary swap

File: api/src/utilities/seed.js [hunk 3]
```diff
@@ -66,9 +67,9 @@ const createPublication = async (title, content) => {
    	        language: 'en',
             licence: 'CC_BY',
             content: content || title,
-            description: title || ''
+            description: title.substr(0,160) || ''
```
Comment (karmseveeratjisc) [inline api/src/utilities/seed.js:70]: this was juSt to get more publications uploading, we want to swap it to 'content' instead

File: api/prisma/seed.ts [hunk 4]
```diff
@@ -18,36 +18,9 @@ export const initialDevSeed = async (): Promise<void> => {
         });
     }
 
-    // Create publications
```
Comment (karmseveeratjisc) [inline api/prisma/seed.ts:21]: again temporary

File: seed-data-scripts/get-errors.js [hunk 5]
```diff
@@ -0,0 +1,92 @@
+const axios = require('axios');
```
Comment (florin-holhos) [inline seed-data-scripts/get-errors.js:1]: all these scripts should be in their own directories with their own package.json files so you can simply install all the required node modules and run the script in it's own directory, independently.

File: seed-data-scripts/seed.js [hunk 6]
```diff
@@ -0,0 +1,362 @@
+const axios = require('axios');
```
Comment (florin-holhos) [inline seed-data-scripts/seed.js:1]: same as the other script, it should stay in it's own directory.

File: seed-data-scripts/seed.js [hunk 7]
```diff
@@ -0,0 +1,362 @@
...
+const titleIDStore = new Map();
+// Manually create the GOD problem via seed data per env then update the DOI here
+titleIDStore.set(
+    'What makes everything we can detect in the universe around us the way that it is, and why?', // GOD problem text
+    'why' // GOD problem DOI
+);
+
+const godProblemTitle = 'What makes everything we can detect in the universe around us the way that it is, and why?';
+
+const tsv = fs.readFileSync('./initialdata.txt', 'utf-8');
```
Comment (florin-holhos) [inline seed-data-scripts/seed.js:13]: what is "tsv" ?

File: seed-data-scripts/get-errors.js (hunk 5 above)
Comment (karmseveeratjisc) [reply seed-data-scripts/get-errors.js:1]: ah yes I simply moved them from api/src/utilities

File: seed-data-scripts/seed.js [hunk 8]
```diff
@@ -0,0 +1,362 @@
...
+        index++;
+
+        try {
+            // Skip header of csv
+            if (index === 0) {
+                continue;
+            }
+
+            const splitRow = row.split('\t');
+            let [title, parent1, parent2, content] = splitRow;
```
Comment (florin-holhos) [inline seed-data-scripts/seed.js:28]: This seems to be very specific to this set of data. Can't there be more than 2 parents? I wonder if this script will work for other similar sets of data

Comment (karmseveeratjisc) [reply seed-data-scripts/seed.js:28]: currently that's the format Alex is supplying in, we won't be running this script again for some time after these sets of data

File: seed-data-scripts/seed.js [hunk 9]
```diff
@@ -0,0 +1,362 @@
...
+            // If there happens to be an error, it's added to errors.txt with the error code
+            // and the row that is occured on.
+            logError(err + ': ' + row);
+            break;
+        }
+    }
+};
+
+const url = process.env.API_URL;
+const apiKey = process.env.APY_KEY;
```
Comment (florin-holhos) [inline seed-data-scripts/seed.js:127]: "API_KEY"

**Review by florin-holhos — DISMISSED** [review]

File: seed-data-scripts/seed.js (hunk 6 above)
Comment (karmseveeratjisc) [reply seed-data-scripts/seed.js]: done

File: seed-data-scripts/get-errors.js (hunk 5 above)
Comment (karmseveeratjisc) [reply seed-data-scripts/get-errors.js]: done

File: seed-data-scripts/seed.js (hunk 9 above)
Comment (karmseveeratjisc) [reply seed-data-scripts/seed.js]: hmm how did that happen, will fix

Comment (karmseveeratjisc) [reply seed-data-scripts/seed.js]: fixed

**Review by florin-holhos — APPROVED** [review]

**Pull request merged**